cabal-install/src/Distribution/Client/Init/NonInteractive/Command.hs

```diff
@@ -276,12 +276,20 @@ licenseHeuristics flags = getLicense flags $ guessLicense flags
 -- | The author's name. Prompt, or try to guess from an existing
 --   darcs repo.
```

Comment:
This could be updated.

Reply:
Yeah, the `darcs repo` part in particular, I guess.